apps/src/signUpFlow/LoginTypeSelection.tsx

```diff
@@ -70,6 +70,24 @@ const LoginTypeSelection: React.FunctionComponent = () => {
     getToken();
   }, []);
 
+  useEffect(() => {
+    const handleKeyDown = (event: KeyboardEvent) => {
+      if (event.key === 'Enter') {
+        const button = document.getElementById(
+          'createAccountButton'
+        ) as HTMLButtonElement;
+        if (button && !button.disabled) {
+          button.click();
+        }
+      }
+    };
+
+    document.addEventListener('keydown', handleKeyDown);
+    return () => {
+      document.removeEventListener('keydown', handleKeyDown);
+    };
+  }, []);
```

Comment:
Love the event listener cleanup here! 

I think there's a more react-y way to do this now. the `TextField` component recently added support for an `onKeyDown` handler. and I think you can call `submitLoginType` directly without having to get the button by id

Reply:
Would an onKeyDown require the focus state to be on that input though? I believe the intention was to allow for an 'enter' to submit if the button is enabled from anywhere on the page

Reply:
Ahh or I would just add it to all three inputs?

Reply:
oh interesting! yeah my thought was to add it to each of the inputs. I'm trying to think of a scenario where the user completes the form and then moves focus away from the buttons and inputs, and then presses enter 🤔 

It's certainly possible, but I'm wondering how necessary it is from an accessibility point of view?

Reply:
in fact, what happens if a user using a screen reader completes the form but then tabs to a link or another button and hits enter?

Reply:
I agree that the three inputs specifically is probably the right move (and have it working, though I cannot for the life of me get this test to work with a focus state).

Reply:
Fixed!